webextensions/api/scripting.json

```diff
@@ -118,7 +119,8 @@
                 "firefox_android": "mirror",
                 "opera": "mirror",
                 "safari": {
-                  "version_added": false
+                  "version_added": "16.4",
+                  "notes": "Available for use in Manifest V2 or later."
```

Comment:
@kiaraarose It appears that the world parameter is still marked as unsupported by Safari.

Reply:
Hi there -- please open a [new issue](https://github.com/mdn/browser-compat-data/issues/new/choose) instead of commenting on an already-merged PR.  This way, we can keep track of the issue and resolve it much quicker!